engine/render/src/render/render_private.h

```diff
@@ -253,6 +253,8 @@ namespace dmRender
     void RenderTypeDebugBegin(HRenderContext rendercontext, void* user_context);
     void RenderTypeDebugDraw(HRenderContext rendercontext, void* user_context, RenderObject* ro, uint32_t count);
 
+    void SetNamedConstantAtIndex(HNamedConstantBuffer buffer, dmhash_t name_hash, dmVMath::Vector4 value, uint32_t value_index);
```

Comment:
Move to DMSDK?

Reply:
Maybe also rename it to SetNamedConstantValueAtIndex or something, since it's used to set a value somewhere in a constant array 🤔